test/test_transforms_v2.py

```diff
@@ -6733,6 +6742,125 @@ def test_functional_error(self):
             F.pil_to_tensor(object())
 
 
+@pytest.mark.skipif(not CVCUDA_AVAILABLE, reason="test requires CVCUDA")
+@pytest.mark.skipif(not CUDA_AVAILABLE, reason="test requires CUDA")
+class TestToCVCUDATensor:
+    """Tests for to_cvcuda_tensor function following patterns from TestToPil"""
+
+    @pytest.mark.parametrize("dtype", [torch.uint8, torch.uint16, torch.float32, torch.float64])
+    def test_1_channel_to_cvcuda_tensor(self, dtype):
+        # Create tensor on CPU first, then move to CUDA to avoid CUDA context issues
+        if dtype in (torch.uint8, torch.uint16):
+            img_data = torch.randint(0, 256, (1, 4, 4), dtype=dtype)
+        else:
+            img_data = torch.rand(1, 4, 4, dtype=dtype)
+        img_data = img_data.cuda()
+        cvcuda_img = F.to_cvcuda_tensor(img_data)
+        assert cvcuda_img is not None
+
+    @pytest.mark.parametrize("dtype", [torch.uint8, torch.uint16, torch.float32, torch.float64])
+    def test_3_channel_to_cvcuda_tensor(self, dtype):
+        # Create tensor on CPU first, then move to CUDA to avoid CUDA context issues
+        if dtype in (torch.uint8, torch.uint16):
+            img_data = torch.randint(0, 256, (3, 4, 4), dtype=dtype)
+        else:
+            img_data = torch.rand(3, 4, 4, dtype=dtype)
+        img_data = img_data.cuda()
+        cvcuda_img = F.to_cvcuda_tensor(img_data)
+        assert cvcuda_img is not None
+
+    def test_invalid_input_type(self):
+        with pytest.raises(TypeError, match=r"pic should be `torch.Tensor`"):
+            F.to_cvcuda_tensor("invalid_input")
+
+    def test_invalid_dimensions(self):
+        # Test 1D array (too few dimensions)
+        # Create tensor on CPU first, then move to CUDA to avoid CUDA context issues
+        with pytest.raises(ValueError, match=r"pic should be 3 or 4 dimensional"):
+            img_data = torch.randint(0, 256, (4,), dtype=torch.uint8)
+            img_data = img_data.cuda()
+            F.to_cvcuda_tensor(img_data)
+
+        # Test 2D array (no longer supported)
+        with pytest.raises(ValueError, match=r"pic should be 3 or 4 dimensional"):
+            img_data = torch.randint(0, 256, (4, 4), dtype=torch.uint8)
+            img_data = img_data.cuda()
+            F.to_cvcuda_tensor(img_data)
+
+        # Test 5D array (too many dimensions)
+        with pytest.raises(ValueError, match=r"pic should be 3 or 4 dimensional"):
+            img_data = torch.randint(0, 256, (1, 1, 3, 4, 4), dtype=torch.uint8)
+            img_data = img_data.cuda()
+            F.to_cvcuda_tensor(img_data)
+
+    @pytest.mark.parametrize("num_channels", [1, 3])
```

Comment:
CV-CUDA can support 2/4-channel tensors, I missed this previously when discussing the validate format function. Same comment for `test_round_trip_batched`.